pkg/config/test_config/db_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
```

Comment:
Should this be db_test, and in pkg/db?

Reply:
if this file in pkg/db will cause import cycle. so put it into pkg/config/test_config